### DISCUSSION TITLE
Construct custom mapper "Full name" for SAML client

### DISCUSSION THREAD

**Discussion by igor-nikiforov** [keycloak/keycloak#9375] [Q&A]
Hello,

I would like to build mapper "Full name" which will based on two existing mappers - `X500 givenName` and `X500 surname`. Could anyone tell me how to achieve this using Custom mapper in SAML client settings?

Thanks.

**Comment by tacerus** [comment]
I'd be interested in this as well.

**Comment by darius-m** [comment]
I am not sure if there are any better alternatives, but you could try to create a Javascript mapper that concatenates the desired user fields. These may not be the proper functions / field names (please use the correct ones if not), but something like the following should work well enough in most cases:
```javascript
user.getGivenName() + " " + user.getSurname()
```
